engine/model.py

```diff
@@ -24,6 +24,8 @@
 # The distance in pixels from the center of the player object that we use for testing
 # activions.
 HITBOX_DISTANCE = 32
+HITBOX_WIDTH = 64
+HITBOX_HEIGHT = 32
```

Comment:
Why wouldn't it be square?

Reply:
I assumed it might be useful to have it be a rectangle, with more control over the hitbox.
If it needs to be a square, is it alright to just use HITBOX_DISTANCE, making the hitbox a 64x64 square in this case for example?